src/modules/bookmarks/components/BookmarkCounter.tsx

```diff
@@ -0,0 +1,34 @@
+/** ****************************************************************************
+ * Copyright 2020-2020 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************** */
+
+import { observer } from 'mobx-react-lite';
+import { useBookmarksStore } from '../../../hooks/useBookmarksStore';
+import 'styles/bookmarks.scss';
+
+// TODO
+// check, what happens, if i don't wrap in observer
```

Comment:
dont commit this kind of comments.

any component that uses mobx stores should be wrapped in observer